
### Kinematic configuration parameter

# Debug mode
activate_output: false
update_rate: 50
# joint_names

# Kinematic parameters
robot_base_link:

tracking_frame:


# limit parameter
min_position_limit: -3.14
max_position_limit:  3.14
min_velocity_limit:  0.0
max_velocity_limit:  2.0
desired_velocity: 1.5

position_tolerance: 0.05
velocity_tolerance: 0.01

# Discretization steps
min_discretization_steps: 10
max_discretization_steps: 20
discretization_steps: 15

